src/main/java/com/easyrest/components/waiter/Order.java

```diff
@@ -0,0 +1,17 @@
+package com.easyrest.components.waiter;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Order {
+    private final WebDriver driver;
```

Comment:
should it be final?

Reply:
not required but can be done, constructor initialization and no reassign